build.gradle

```diff
@@ -927,69 +425,35 @@ publishing {
 			}
 			artifact sourcesJar
 			artifact javadocJar
-		}
 
+			pom.withXml {
+				def hashedDep = it.asNode().appendNode('dependencies').appendNode('dependency')
+				hashedDep.appendNode('groupId', 'org.quiltmc')
+				hashedDep.appendNode('artifactId', 'hashed')
+				hashedDep.appendNode('version', configurations.hashed.getDependencies().iterator().next().getVersion())
```

Comment:
While we still have Hashed in the snapshot maven, this should point to the resolved hashed version, ie: `21w41a-20211013.160757-1`

Reply:
This snippet should work
```groovy
import org.gradle.api.internal.artifacts.repositories.resolver.MavenUniqueSnapshotComponentIdentifier
// ...
def id = configurations.hashed.getResolvedConfiguration().getResolvedArtifacts().iterator().next().getId().getComponentIdentifier()
def version
if (id instanceof MavenUniqueSnapshotComponentIdentifier) {
	version = id.getVersion().toString().replace("SNAPSHOT", id.getTimestamp())
} else {
	version = id.getVersion().toString()
}
```